activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   `AR::Relation#reverse_order` throws `AR::IrreversableOrderError` when the order can not be reversed
```

Comment:
sorry but this is still typo should be Irrevers**i**ble, not Irrevers**a**ble :-)